r2dbc-mysql/src/main/java/SimpleStatement.kt

```diff
@@ -12,6 +13,7 @@ class SimpleStatement(private val clientSupplier: Supplier<JasyncConnection>, pr
 
     private var isPrepared = false
     private var params: MutableMap<Int, Any?> = mutableMapOf()
+    private var selectLastInsertId: Boolean = false
```

Comment:
We should also add `private var generatedKeyName: String = "LAST_INSERT_ID"` here.

Reply:
added below